docs/source/training_guide.rst

```diff
@@ -0,0 +1,80 @@
+models more efficiently and with less data.
+
+Since the process of fine-tuning models can be tricky, this section of the docs provides some guidelines and insights that may 
+be useful in order to succesfully adapt our models. See :ref:`fine_tuning` to view an example of
+how to retrain our models.
+
+.. note::
+   We provide practical recommendations for the fine-tuning process, but you may have to try multiple retraining configurations to achieve an optimal result. If you are having difficulty adapting our models to your use-case, feel free to
+   open an issue on the deepparse `Github <https://github.com/GRAAL-Research/deepparse/issues>`_.
+
+There are a few use-cases that may lead you to want to retrain deepparse's models. Whether you wish to obtain a better 
+performance on a single or multiple countries that our models weren't trained on, or your data and address schemes require a more complex 
+architecture, or the tag structure of your dataset is different from ours; deepparse's retraining features accomodate all these use-cases, and more.
+
+In practice, our models were trained on 20 countries and demonstrated very accurate results on all of them, so unless you wish to predict 
+different tags (e.g: StreetNumber...) we advise you to use our models without retraining. Also, if you wish to retrain 
+our models to obtain a better performance on countries outside of the 20 used in the original training set, you can take a look 
+at `our dataset <https://github.com/GRAAL-Research/deepparse-address-data>`_ which includes an additional 41 countries that were only used for testing.
+
+There are two main concerns to keep in mind when fine-tuning a model: the model's convergence (i.e: its ability to actually learn from the new data) 
+and the possibility of `catastrophic forgetting` (i.e: losing the model's previous knowledge after training on the new data).
+
+Learning successfully
+*********************
+
+Making a model converge is as much an art as it is a science since it often requires quite a bit of experimentation and parameter tuning. In the case 
+of fine-tuning, the models have already developed a base knowledge of the task that they were trained on, which gives them an edge.
+This is especially true in the case of deepparse, since the task you are fine-tuning on remains exactly the same (i.e: parsing addresses). 
+However, there are a couple of points to consider in order to obtain favourable results:
+
+- **Make sure you have enough data**: deep learning models are notorious for being pretty data hungry, so unless you have enough data the models 
+  are going have a hard time learning. Since deepparse's models have already been trained on a few million addresses, the need for data is mitigated when it come to fine-tuning. However, it is recommended to at least have a few thousand examples per new country when retraining.
```

Comment:
```suggestion
  are going have a hard time learning. Since Deepparse's models have already been trained on a few million addresses, the need for data is mitigated when it comes to fine-tuning. However, it is recommended to use at least a few thousand examples per new country when retraining.
```